regtests/ww3_tp2.22/input/ww3_ounf.inp

```diff
@@ -0,0 +1,43 @@
+$ -------------------------------------------------------------------- $
+$ WAVEWATCH III Grid output post-processing                            $
+$--------------------------------------------------------------------- $
+$ First output time (yyyymmdd hhmmss), increment of output (s), 
+$ and number of output times.
+$
+  19680606 000000 10800. 100
+$
+$ Fields requested --------------------------------------------------- $
+$
+$ Output request flags identifying fields as in ww3_shel.inp. See that
+$ file for a full documentation of field output options. Namelist type
+$ selection is used here (for alternative F/T flags, see ww3_shel.inp).
+$
+ N
+ HS T01 DIR SPR
```

Comment:
The output is to add things to stoke's drift, but I don't see us outputting stoke's drift here, perhaps this should be added to properly test the new features?